nvflare/private/fed/app/client/client_train.py

```diff
@@ -56,6 +57,7 @@ def main():
     args.env = os.path.join("config", AppFolderConstants.CONFIG_ENV)
     args.train_config = os.path.join("config", AppFolderConstants.CONFIG_TRAIN)
     args.log_config = None
+    args.job_id = None
```

Comment:
why do we need the job_id here?

Reply:
when creating the cell, it uses the job_id to distinguish creating cell for different client parent and child.